pytravharv/WebAccess.py

```diff
@@ -43,7 +44,7 @@ def handle_data(self, data):
             self.scripts.append({self.type: data})
 
 
-def WebAccess(url):
+def fetch(url):
     triplestore = Graph()
```

Comment:
`triplestore` name slightly confusing? both ends know and expect an `rdflib.Graph`, no?

just `g` would do for me here